src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1173,28 +1173,25 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, BindingDiagnost
             }
             else
             {
+                var boundTypeArguments = BindTypeArguments(typeArguments, diagnostics, basesBeingResolved);
+                if (unconstructedType.IsGenericType
+                    && options.IsAttributeTypeLookup()
+                    && boundTypeArguments.FirstOrDefault(bta => bta.Type.IsTypeParameter() || bta.Type.IsUnboundGenericType() || bta.Type.ContainsTypeParameter()) is { HasType: true } badAttributeArgument)
```

Comment:
>bta.Type [](http://example.com/codeflow?start=129&length=8)

Can we reach here with `bta.Type is null`? For instance `[MyAttribute<System>]`.

Reply:
I do not think so. Adding a test.